community/kernel/src/main/java/org/neo4j/kernel/api/index/IndexAccessor.java

```diff
@@ -90,6 +91,17 @@
      */
     ResourceIterator<File> snapshotFiles() throws IOException;
 
+    /**
+     * Verifies that each value in this index is unique.
+     * Index is guaranteed to not change while this call executes.
+     *
+     * @param propertyAccessor {@link PropertyAccessor} for accessing properties from database storage
+     * in the event of conflicting values.
+     * @throws IndexEntryConflictException for first detected uniqueness conflict, if any.
+     * @throws IOException on error reading from source files.
+     */
+    void verifyDeferredConstraints( PropertyAccessor propertyAccessor ) throws IndexEntryConflictException, IOException;
```

Comment:
Yes, although I opted to keep those two methods the same to not introduce confusion. I donät think it makes sense to name them differently and I wanted to keep changes small due to being in a maintenance branch.